app/presenters/schools/users/show_presenter.rb

```diff
@@ -0,0 +1,112 @@
+module Schools
+  module Users
+    class ShowPresenter < ApplicationPresenter
+      attr_reader :user
+
+      def initialize(view_context, user)
+        @user = user
+        super(view_context)
+      end
+
+      def avatar_url
+        user.avatar_url || user.initials_avatar
+      end
+
+      def courses_taken
+        @courses_taken ||=
+          begin
+            Course
+              .joins(cohorts: { students: :user })
+              .where(users: { id: user.id })
+              .order(name: :asc)
+              .distinct
+          end
+      end
+
+      def courses_coached
+        @courses_coached ||=
+          begin
+            Course
+              .joins(cohorts: { faculty: :user })
+              .where(users: { id: user.id })
+              .order(name: :asc)
+              .distinct
+          end
+      end
+
+      def courses_authored
+        @courses_authored ||=
+          begin
+            Course
+              .joins(course_authors: :user)
+              .where(users: { id: user.id })
+              .order(name: :asc)
+          end
+      end
+
+      def organisation_names
+        @organisation_names ||= user.organisation&.name
+      end
+
+      def current_standing
+        @current_standing ||=
+          begin
+            user
+              .user_standings
+              .includes(:standing)
+              .live
+              .order(created_at: :desc)
+              .first
+              &.standing || current_school.default_standing
+          end
+      end
+
+      def type_tags
+        @type_tags ||=
+          begin
+            tags = []
+            tags << "Admin" if user.school_admin.present?
+            tags << "Student" if courses_taken.present?
+            tags << "Coach" if courses_coached.present?
+            tags << "Author" if courses_authored.present?
+
+            tags.join(" • ")
+          end
+      end
```

Comment:
These are text elements that show up in the view, right? This approach doesn't support i18n.